src/OmnichannelChatSDK.ts

```diff
@@ -1294,7 +1294,8 @@ class OmnichannelChatSDK {
             const uploadDocumentResponse: any = await this.AMSClient?.uploadDocument(documentId, fileInfo as any);  // eslint-disable-line @typescript-eslint/no-explicit-any
 
             const fileIdsProperty = {
-                amsReferences: JSON.stringify([documentId])
+                amsReferences: JSON.stringify([documentId]),
+                amsreferences: JSON.stringify([documentId])
```

Comment:
why is needed to pass exactly the same value twice in the same objecT?

Reply:
It's for backward compatibility. It will be eventually removed once all the fixes will be in production.